lte/gateway/python/integ_tests/s1aptests/s1ap_utils.py

```diff
@@ -1943,15 +2077,15 @@ def _print_directoryd_content(self):
             )
         except grpc.RpcError as e:
             print(
-                "error: couldnt print directoryd content. gRPC failed with %s: %s"
-                % (e.code(), e.details()),
+                f"error: couldnt print directoryd content. "
+                f"gRPC failed with {err.code()} {err.details()}",
             )
```

Comment:
You renamed `e` to `err` in the print statement but not in the `except` clause `except grpc.RpcError as e:`